src/applications/gi/reducers/calculator.js

```diff
@@ -128,6 +134,97 @@ export default function (state = INITIAL_STATE, action) {
         ...newState
       };
     }
+    case FETCH_BAH_FAILED: {
+      const { beneficiaryZIPFetched } = action;
+      const error = _.get(action, 'payload.errors[0].title');
+
+      // institution and zipcode_rates endpoints both return this generic error
```

Comment:
I don't like this approach to error handling but probably the best we can do right now. We should probably report other kinds of errors but I don't think this app has any error reporting.